### PR TITLE
Drop PHP 7, Support PHP 8.2

### PR DESCRIPTION
**Pull request by rogervila**
Signed-off-by: Roger Vilà <rogervila@me.com>

|    Q          |   A
|-------------- | ------
| Documentation | no
| Bugfix        | no
| BC Break      | no
| New Feature   | no
| RFC           | no
| QA            | no

### Description

Drop PHP 7, Support PHP 8.2

### CODE REVIEW

File: src/AssetUninstaller.php [hunk 1]
```diff
@@ -104,7 +104,10 @@ public function __invoke($package) /*PackageInterface*/
             return;
         }
 
+        /** @psalm-var array{asset_manager: array{resolver_configs: array{paths: string[]}}} */
         $packageConfig = include $packageConfigPath;
+
+        /** @psalm-suppress DocblockTypeContradiction */
```
Comment (Ocramius) [inline src/AssetUninstaller.php]: Let's expand the baseline instead

File: test/PluginTest.php [hunk 2]
```diff
@@ -121,6 +121,7 @@ public function testOnPostAutoloadDumpTriggersInstallers(): void
 
         $this->assertNull($plugin->onPostAutoloadDump());
 
+        /** @psalm-suppress TypeDoesNotContainType */
```
Comment (Ocramius) [inline test/PluginTest.php]: Let's expand the baseline instead

**Review by Ocramius — CHANGES_REQUESTED** [review]

File: src/AssetUninstaller.php (hunk 1 above)
Comment (rogervila) [reply src/AssetUninstaller.php]: Done

File: test/PluginTest.php (hunk 2 above)
Comment (rogervila) [reply test/PluginTest.php]: Done

**Review by Ocramius — APPROVED** [review]
Thanks @rogervila!

**Pull request merged**